packages/ckeditor5-dev-bundler-webpack/lib/utils/getwebpackconfig.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const webpack = require( 'webpack' );
+const getWebpackES6Config = require( './getwebpackes6config' );
+
+/**
+ * @param options
+ * @param {String} options.entryPoint An entry point which will be compiled.
+ * @param {String} options.destinationPath A path where compiled file will be saved.
+ * @param {String} options.moduleName A name of exported module after compilation.
+ * @param {String} options.cwd Current work directory. Required for searching the modules.
+ * @returns {Object}
+ */
+module.exports = function getWebpackConfig( options ) {
+	const config = getWebpackES6Config( options );
```

Comment:
I don't like that one config is an extension of the other one because it's not. See that you're overriding the `plugins` property, so it's not an inheritance and it shouldn't be implemented here as such.

E.g. what if the ES6 config will require more plugins? You don't know if you need to override just Babili or all of them.

So, let's KISS here and just duplicate the config object twice. After all – "Duplication is easier to fix than broken abstraction"